src/test/exporter/promethize_data.h

```diff
@@ -0,0 +1,655 @@
+#pragma once
+
+#include <string>
+#include <vector>
+
+using namespace std;
```

Comment:
`using namespace` directive in header files is a bad idea and should be avoided, especially if it's for `std`.

Taking a step back, why is this in a header file to begin with? Do you expect promethize_data vector to be used by something other than Exporter.promethize test case?

Reply:
Removed the using std. 

I moved this to a header file to remove potential bloat for the unittest file of the exporter.

Reply:
> I moved this to a header file to remove potential bloat for the unittest file of the exporter.

Since it's getting compiled for the unittest (in the same translation unit, and _only_ in that translation unit), I don't think that is a valid reason for creating a header file with `#pragma once`, etc.  Moving over a definition like in the source file takes one or two keystrokes in any decent editor.